lib/wifi_user/use_case/email_sponsees_extractor.rb

```diff
@@ -1,14 +1,16 @@
 require 'nokogiri'
 
 class WifiUser::UseCase::EmailSponseesExtractor
-  def initialize(email_fetcher:)
+  def initialize(email_fetcher:, sponsor_address:)
```

Comment:
to keep this class contained, might be worth renaming this to be more generalised:

`exluded_sponsees`